python/activator/middleware_interface.py

```diff
@@ -45,6 +45,35 @@
 _log.setLevel(logging.DEBUG)
 
 
+def get_central_butler(central_repo: str, instrument_class: str):
+    """Provide a useful Butler that can access the given repository and work on
+    data for the given instrument.
+
+    Parameters
+    ----------
+    central_repo : `str`
+        The path or URI to the central repository.
+    instrument_class : `str`
+        The name of the instrument whose data will be retrieved. May be either
+        the fully qualified class name or the short name.
+
+    Returns
+    -------
+    butler : `lsst.daf.butler.Butler`
+        A Butler for ``central_repo`` pre-configured to load
+        ``instrument_name`` data.
+    """
+    # TODO: how much overhead do we take on from asking the central repo about
+    # the instrument instead of handling it internally?
+    registry = Butler(central_repo).registry
+    instrument = lsst.obs.base.Instrument.from_string(instrument_class, registry)
+    return Butler(central_repo,
+                  collections=[instrument.makeCollectionName("defaults")],
+                  writeable=True,
```

Comment:
Why does this one need to be writeable; isn't this just for accessing calibs and templates? If it really does, that should probably be noted in the Returns comment.

Reply:
Good catch; I'd written the docs before DM-35053 merged and forgot to update them.